test/e2e/framework/addon/vault/vault.go

```diff
@@ -40,9 +40,9 @@ import (
 
 const (
 	vaultHelmChartRepo    = "https://helm.releases.hashicorp.com"
-	vaultHelmChartVersion = "0.22.1"
-	vaultImageRepository  = "index.docker.io/library/vault"
-	vaultImageTag         = "1.12.1@sha256:08dd1cb922624c51a5aefd4d9ce0ac5ed9688d96d8a5ad94664fa10e84702ed6"
+	vaultHelmChartVersion = "0.24.1"
+	vaultImageRepository  = "local/vault"
```

Comment:
I guess alternatively we could get rid of the retag make target and pass the image version/sha as an env var into the tests. However, that'd be a bit more work- this seems like an improvement as at least we aren't defining the SHA in two places 👍🏼